app/src/pages/inside/common/statusDropdown/statusDropdown.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import className from 'classnames/bind';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import { activeProjectSelector } from 'controllers/user';
+import { fetchTestItemsAction } from 'controllers/testItem';
+import { NOTIFICATION_TYPES, showNotification } from 'controllers/notification';
+import { fetch } from 'common/utils/fetch';
+import { URLS } from 'common/urls';
+import { InputDropdown } from 'components/inputs/inputDropdown';
+import { formatStatus } from 'common/utils/localizationUtils';
+import { PASSED, FAILED, SKIPPED, INTERRUPTED, IN_PROGRESS } from 'common/constants/testStatuses';
+import styles from './statusDropdown.scss';
+
+const cx = className.bind(styles);
+
+const messages = defineMessages({
+  itemUpdateSuccess: {
+    id: 'EditItemsModal.itemUpdateSuccess',
+    defaultMessage: 'Completed successfully!',
+  },
+});
+
+@connect(
+  (state) => ({
+    currentProject: activeProjectSelector(state),
+  }),
+  {
+    showNotification,
+    fetchTestItemsAction,
+  },
+)
+@injectIntl
+export class StatusDropdown extends Component {
+  static propTypes = {
+    data: PropTypes.shape({
+      itemId: PropTypes.number,
+      status: PropTypes.string,
+      attributes: PropTypes.array,
+      description: PropTypes.string,
+    }).isRequired,
+    currentProject: PropTypes.string.isRequired,
+    intl: intlShape.isRequired,
+    fetchTestItemsAction: PropTypes.func,
+  };
+
+  static defaultProps = {
+    fetchTestItemsAction: () => {},
+  };
+
+  updateItem = (newStatus) => {
+    const {
+      intl: { formatMessage },
+      currentProject,
+      data: { status: oldStatus, itemId, attributes, description },
+    } = this.props;
+    const data = {
+      attributes,
+      description,
+      status: newStatus,
+    };
+
+    if (newStatus === oldStatus) return;
+
+    fetch(URLS.testItemUpdate(currentProject, itemId), { method: 'put', data }).then(() => {
+      showNotification({
+        message: formatMessage(messages.itemUpdateSuccess),
+        type: NOTIFICATION_TYPES.SUCCESS,
+      });
+      this.props.fetchTestItemsAction();
+    });
+  };
+
+  generateOptions = (status) => {
+    const { intl } = this.props;
+    if (status === IN_PROGRESS)
+      return [{ label: formatStatus(intl.formatMessage, IN_PROGRESS), value: IN_PROGRESS }];
+
+    const STATUS_TYPES = [PASSED, FAILED, SKIPPED];
+    if (status === INTERRUPTED) STATUS_TYPES.push(INTERRUPTED);
+
+    return STATUS_TYPES.map((item) => ({
+      label: formatStatus(intl.formatMessage, item),
+      value: item,
+    }));
+  };
+
+  render() {
+    const {
+      data: { status },
+    } = this.props;
+    return (
+      <div className={cx('status-dropdown')}>
+        <InputDropdown
```

Comment:
Please try using `components/inputs/inputDropdownSorting` - it looks like the dropdown you need here

Reply:
Yes, `InputDropdownSorting` looks like dropdown I need here, but has another arrow shape and behavior, so when trying to use it I should modify not only styles, but also code inside this component.

Reply:
Could you give some examples?

Reply:
```
        
![Screen Shot 2019-08-02 at 12 52 54 PM](https://user-images.githubusercontent.com/34836029/62361874-8d903880-b524-11e9-84fe-ef9699b2885f.png)
<InputDropdownSorting
          value={status}
          options={this.generateOptions(status)}
          onChange={this.updateItem}
          disabled={status === IN_PROGRESS}
          mobileDisabled
          transparent
        />
```

Reply:
![Screen Shot 2019-08-02 at 12 52 54 PM](https://user-images.githubusercontent.com/34836029/62361978-b7e1f600-b524-11e9-9024-2c24ef2c4078.png)

Reply:
![Screen Shot 2019-08-02 at 12 57 24 PM](https://user-images.githubusercontent.com/34836029/62362179-1c9d5080-b525-11e9-87fa-a76af885d91e.png)